x-pack/solutions/security/packages/kbn-securitysolution-utils/src/path_validations/index.ts

```diff
@@ -25,9 +25,11 @@ export enum ConditionEntryField {
 export enum TrustedDeviceConditionEntryField {
   USERNAME = 'user.name',
   HOST = 'host.name',
-  DEVICE_ID = 'device.id',
-  MANUFACTURER = 'device.manufacturer',
-  PRODUCT_ID = 'device.model.identifier',
+  DEVICE_ID = 'device.serial_number',
```

Comment:
For external EDRs I think we're using `device.id` for consistency. Just noting here in case it matters ;)